### PR TITLE
Update mentor model migrations

### PR DESCRIPTION
**Pull request by hodgestar**
Currently the migrations for the mentor app are lagging behind the code base. We need to update them once #87 lands.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: mentorship/comms/tests/test_comms_views.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def test_call_feedback_handler_onion_dev(self):
         call.ended()
         url = reverse(self.url_name_base + 'onion-dev-feedback')
         data = {
-            "cdr": {"callerid": "1234"},
+            "cdr": {"callerid": "1234"}, "complete": True,
```
Comment (bruskiza) [inline mentorship/comms/tests/test_comms_views.py]: What states can these be in? Complete, not complete or unknown?

Comment (hodgestar) [inline mentorship/comms/tests/test_comms_views.py]: You probably want to review #86 first (it contains the documentation for this).

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**